core/include/logger/logger.h

```diff
@@ -64,26 +64,51 @@ class Logger {
   /*                API                */
```

Comment:
Is the Logger MIT license? If the license must change, it should happen in both `logger.h` and `logger.cc`